Gemfile

```diff
@@ -1,11 +1,16 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
-git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+git_source(:github) { |_repo| 'https://github.com/#{repo}.git' }
 
+# Specify your gem's dependencies in camerata.gemspec
+gemspec
+
+# TODO: make these development dependencies?
```

Comment:
Thinking about that question -- the question in my mind is, is there a use case where people will be wanting to use the camerata gem but not have the smoke tests available?

Reply:
this makes a lot of sense to me.